hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -526,7 +540,7 @@ async def _create_order(
             api_params = {
                 "id": order_uuid,
                 "time": timestamp,
-                "symbol": await OBDS.exchange_symbol_associated_to_pair(trading_pair),
+                "symbol": await AscendExAPIOrderBookDataSource.exchange_symbol_associated_to_pair(trading_pair),
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols